pkgs/development/libraries/sgp4/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, stdenv, fetchFromGitHub, cmake, pkg-config } :
+
+stdenv.mkDerivation {
+  pname = "sgp4";
+  version = "20210111";
+
+  src = fetchFromGitHub {
+    owner = "dnwrnr";
+    repo = "sgp4";
+    rev = "ca9d4d97af4ee62461de6f13e0c85d1dc6000040";
+    sha256 = "sha256-56It/71R10U+Hnhw2tC16e5fZdyfQ8DLx6LVq65Rjvc=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config ];
```

```suggestion
  nativeBuildInputs = [ cmake ];
```
There are no libraries to find.